src/testing/packet_simulator.zig

```diff
@@ -257,8 +262,12 @@ pub fn PacketSimulatorType(comptime Packet: type) type {
                 var to: u8 = 0;
                 while (to < self.node_count()) : (to += 1) {
                     const path = .{ .source = from, .target = to };
-                    self.links[self.path_index(path)].enabled = from >= self.options.replica_count or
-                        to >= self.options.replica_count or partition[from] == partition[to];
+                    self.links[self.path_index(path)].enabled =
+                        from >= self.options.replica_count or
+                        to >= self.options.replica_count or
+                        partition[from] == partition[to] or
+                        (self.options.partition_symmetry == .asymmetric and
+                        partition[from] == random.boolean());
```

Comment:
This simplifies to 

```suggestion
                        (self.options.partition_symmetry == .asymmetric and random.boolean());
```

?

Perhaps we want to do rather

```
const assymetric_partition_side = random.boolean()
while (to < self.node_count()) : (to += 1) { 
   ...
  (self.options.partition_symmetry == .asymmetric and
                        partition[from] == assymetric_partition_side);
}
```